Automattic/wp-calypso
# Jetpack Cloud: Initiate a scan when the user hits 'Scan Now'

#### Changes proposed in this Pull Request

* Make a call to the API whenever a user clicks the "Scan now" button on the Scanner page, initiating a scan of their site.

#### Testing instructions



* Open Jetpack Cloud and select a site with Scan enabled and no detected threats (i.e., "okay" state).
* Go to the Scanner page and click "Scan now."
* Verify that the UI immediately changes to reflect that a scan has been initiated.
* Wait for a few seconds, then verify that the UI reflects the site's "true" scan state as returned by the API. Requesting a scan may not always result in a scan immediately taking place (e.g., if an error occurs).

Fixes 1151678672052943-as-1171571159576667.
